src/scripts/main.js

```diff
@@ -1 +1,18 @@
 'use strict';
+
+// eslint-disable-next-line no-shadow
+document.addEventListener('click', (event) => {
+  const img = event.target.closest('img');
+
+  if (img) {
+    event.preventDefault();
+
+    const link = img.closest('a');
+
+    if (link) {
+      const largeImg = document.getElementById('largeImg');
+
+      largeImg.src = link.getAttribute('href');
```

Comment:
Consider checking if `largeImg` is not null before setting its `src` attribute. This will prevent potential errors if the element with ID `largeImg` does not exist in the DOM.